commands/pass.go

```diff
@@ -0,0 +1,59 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/bndw/pick/utils/pswdgen"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+func init() {
+	cmd := &cobra.Command{
+		Use:   "pass",
+		Short: "Generate a password without storing it",
+		Long:  "The pass command is used to generate a password without storing it.",
+		Run: func(cmd *cobra.Command, args []string) {
+			runCommand(Pass, cmd, args)
+		},
+	}
+	cmd.Flags().Int("length", pswdgen.DefaultPasswordLength, "the length of the generated password")
```

Comment:
My desire is to use the `pass` command to generate single, one-time use passwords in non-interactive mode. However, when I use the `add` command I prefer interactive mode.

Do you have a suggested configuration/usage to achieve that use case?